bert_e/docs/USER_DOC.md

```diff
@@ -170,6 +171,20 @@ target development branch. This code is then tested in the build pipeline,
 before any merge can happen. There are as many integration branches as there
 are target branches.
 
+The integration branches are mandatory in the GitWaterFlow process,
+and by default they are automatically created when a pull request is
+opened.
+This behaviour can be changed with the `always_create_integration_branches`
+parameter in the bot settings:
+
+* *true* (default): integration branches are created automatically when a
+  pull request is opened.
+* *false*: users will be required to explicitly request the creation
+  of integration branches by adding a `/create_integration_branches`
+  comment in their pull request.
+  The above allow to temporize the creation of those branches as
+  in the review process requested changes as expected.
```

Comment:
Not sure that's relevant here. If we add more commits later to take care of requested changes, then the integration branches will just naturally evolve. Their roles and solve any conflict before moving to the queues, so it makes sense to always have them, unless it's too costly.

Reply:
I disagree with this point; I think it makes sense to keep this sentence. What do you think @tcarmet ?

Reply:
Depends, why do you think it make sense to keep this sentence?

Reply:
It just provides more details on why there is no need to specify `/create_integration_branches` when typing `approve` and `create_pull_requests`.

Reply:
Yes, I had understood correctly. My comment remains the same. However, I will remove it as it may not be clear.